Cargo.toml

```diff
@@ -39,3 +42,24 @@ bevy = { version = "0.10", default-features = false, features = [
     "bevy_pbr",
     "bevy_core_pipeline",
 ] }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+winit = "0.28"
+web-sys = { version = "0.3.63", features = [
+    "Clipboard",
+    "ClipboardEvent",
+    "DataTransfer",
+    'Document',
+    'EventTarget',
+    "Window",
+    "Navigator",
+] }
+js-sys = "0.3.63"
+wasm-bindgen = "0.2.84"
+wasm-bindgen-futures = "0.4.36"
+console_log = "1.0.0"
+log = "0.4"
+crossbeam-channel = "0.5.8"
```

Comment:
maybe we should avoid specifying the last number (omit patch version) ?

Reply:
That shouldn't matter much, as cargo won't prevent users of our crate doing semver compatible updates. I personally prefer omitting the patch part unless there's a hard requirement (e.g. no less than a specific patch version).